packages/react/vitest.config.ts

```diff
@@ -1,6 +1,9 @@
 import { defineConfig } from "vitest/config";
+import { vanillaExtractPlugin } from "@vanilla-extract/vite-plugin";
 
 export default defineConfig({
+  // @ts-ignore
```

Comment:
I'll get to the bottom of this, it compiles after ignoring the error but it looks like there is some mismatch between `vitest` and `vanilla-extract`